composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "neos/neos-ui-compiled": "^3.0 | ^5.0"
+        "neos/neos-ui-compiled": "^3.0 | ^4.0 | ^5.0"
```

Comment:
Shouldn't it be a `||`?
```
"neos/neos-ui-compiled": "^3.0 || ^4.0 || ^5.0"
```

Reply:
Yes please update it to `||`. :) Thank you 👍

Reply:
Wait i will fix it :)